code/game/machinery/cloning.dm

```diff
@@ -279,8 +292,20 @@
 		H.set_cloned_appearance()
 
 		H.set_suicide(FALSE)
+
+
 	attempting = FALSE
-	return CLONING_SUCCESS
+	return result
+
+/obj/machinery/clonepod/proc/offer_to_ghost(mob/living/carbon/H)
+	set waitfor = FALSE
+	var/list/mob/dead/observer/candidates = pollCandidatesForMob("Do you want to play as [H.real_name]'s experimental clone?", ROLE_EXPERIMENTAL_CLONE, null, null, 300, H, POLL_IGNORE_EXPERIMENTAL_CLONE)
+	if(LAZYLEN(candidates))
```

Comment:
candidates is not lazy

Reply:
I just copy-pasted it from others. all other codes do the same thing.

Reply:
length(candidates)

Reply:
I updated it, but not all LAZYLEN(candidates) at other codes since I don't want to test possible bugs as it is not a PR for candidates LAZYLEN fixes.